config/initializers/doorkeeper_openid_connect.rb

```diff
@@ -1,8 +1,14 @@
 Doorkeeper::OpenidConnect.configure do
   issuer ENV['KOALA_DOMAIN']
 
-  # This key should always exists, otherwise OIDC logins are unsecured
-  signing_key File.read(ENV['OIDC_SIGNING_KEY'])
+  if Rails.env.development? then
+    # FIXME: we can use an actual key in development, but just not when building
```

Comment:
Could you explain why the secret key can't be used here?

Reply:
We cannot bake the (fake) secret in the build, because we need an actual secret on the server. Since we host the build in a public place, we cannot bake the actual secret in the build (and it would make rotating the secret much harder too).
The correct solution is to put the fake secret in the `.env` file and revert the change in this file. I probably had my reasons while developing this, but I don't recall exactly.